composer.json

```
@@ -14,7 +14,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.1",
-        "ezsystems/ezpublish-kernel": "^8.0@dev",
+        "ezsystems/ezpublish-kernel": "dev-ezp_30463_solr_cloud as 7.5",
```

```suggestion
        "ezsystems/ezpublish-kernel": "^7.5.2@dev || ^8.0@dev",
```

Once kernel pr is merged.